Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,19 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+// Task1: Display names of people in Gryffindor
+console.log("Gryffindor Members:");
+hogwarts.forEach(({ firstName, lastName, house }) => {
+  if (house === "Gryffindor") {
```

Comment:
Your code works well 👍
Non-blocking suggestion: If you wanted to improve clarity, you can use `.filter` to only include records where house === 'Gryffindor' before running `.forEach`.